### PR TITLE
Update AMIs from 2015.09.b to 2015.09.c.

### LINKED ISSUE #56: EC2 Container Registry Support
Currently, the AMI in the ecs-cli is 2015.03.b, which is the ECS Agent 1.1.0
https://github.com/aws/amazon-ecs-cli/blob/cff682c389b5d863a44f8e38972fd039f4517f9b/ecs-cli/modules/config/ami/ami.go#L29

ECR requires that the ECS Agent 1.7.0 or greater, so we should update for the latest AMI of 2015.03.g.
http://docs.aws.amazon.com/AmazonECS/latest/developerguide/container_agent_versions.html

Upgrading this will allow ECR images to be used with clusters created by the ecs-cli.

### PR DESCRIPTION
**Pull request by skion**
Got these from http://docs.aws.amazon.com/AmazonECS/latest/developerguide/launch_container_instance.html

This should fix #56.

### CODE REVIEW

File: ecs-cli/modules/config/ami/ami.go [hunk 1]
```diff
@@ -29,12 +29,12 @@ type staticAmiIds struct {
 func NewStaticAmiIds() ECSAmiIds {
 	regionToId := make(map[string]string)
 	// amzn-ami-2015.09.b-amazon-ecs-optimized AMIs
```
Comment (samuelkarp) [inline ecs-cli/modules/config/ami/ami.go]: Can you update the comment here?

**Comment by samuelkarp** [comment]
Thanks for doing this!  I believe it should be safe to add the two new regions as well.

**Comment by frewsxcv** [comment]
Just tried to use the ECS Registry with ecs-cli and was sad to see it wasn't natively supported. But glad to see it was addressed in this PR. Thanks @skion! :confetti_ball:

**Comment by frewsxcv** [comment]
The fail on Travis CI seems unrelated, no?

**Comment by euank** [comment]
Reran the failed one; it looked related to a [transient Github issue](https://status.github.com/).

:+1: from me.

**Comment by aaithal** [comment]
lgtm as well

**Comment by samuelkarp** [comment]
:+1:

**Comment by juanrhenals** [comment]
:+1:

**Comment by sentientmonkey** [comment]
Thanks again, Pieter! :beer:

**Comment by skion** [comment]
Thank you for the fast turnover!

**Pull request merged**